app/backend/src/couchers/servicers/auth.py

```diff
@@ -201,9 +201,11 @@ def CompleteSignup(self, request, context):
             if not signup_token:
                 context.abort(grpc.StatusCode.NOT_FOUND, errors.INVALID_TOKEN)
 
-            # should be in YYYY-MM-DD format
+            # check birthdate validity (YYYY-MM-DD format and in the past)
             try:
                 birthdate = datetime.fromisoformat(request.birthdate)
+                if birthdate >= datetime.now():
+                    context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_BIRTHDATE)
```

Comment:
This should be moved outside the try-catch block. The block is there to catch any errors in date formatting. What you've added shouldn't raise a ValueError.

Also, please use the `now()` function from `couchers.utils`. It's timezone aware and should be used everywhere. Timezone bugs are a pain.